.github/workflows/makefile.yml

```diff
@@ -1,10 +1,11 @@
-name: Makefile CI
+name: Update errata
 
 on:
   push:
     branches: [ "master" ]
-  pull_request:
-    branches: [ "master" ]
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
maybe pick a "random" minute and second?

Reply:
@richsalz https://github.com/reschke/rfc-errata/pull/2 Provided that rfc-editors server time is UTC, it has least traffic during 18th hour. :)